build/github-tf-pull-request.yaml

```diff
@@ -12,13 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: 'tf-pull-request'
```

Comment:
why replacing `'` with `"` and why replacing only on this files and not on the [github-tf-apply.yaml](https://github.com/mariammartins/terraform-google-enterprise-genai/blob/74ff4ce23dddacfc70b97aa9a92b9ff49843d60b/build/github-tf-apply.yaml) too ?

Reply:
I didn't found any errors related to the use of `'`, this change was based on the practice we follow in the foundation.

I will standardize the use of `""` in the [github-tf-apply.yaml](https://github.com/mariammartins/terraform-google-enterprise-genai/blob/74ff4ce23dddacfc70b97aa9a92b9ff49843d60b/build/github-tf-apply.yaml) file too.